lib/elixir_internal_certificate_web/controllers/api/v1/user_registration_controller.ex

```diff
@@ -0,0 +1,23 @@
+defmodule ElixirInternalCertificateWeb.Api.V1.UserRegistrationController do
+  use ElixirInternalCertificateWeb, :controller
+
+  alias ElixirInternalCertificate.Account.Accounts
+  alias ElixirInternalCertificate.Account.Schemas.User
+  alias ElixirInternalCertificateWeb.Api.ErrorView
+  alias ElixirInternalCertificateWeb.Api.V1.UserRegistrationView
+
+  def create(conn, params) do
+    with changeset <- Accounts.change_user_registration(%User{}, params),
+         {:ok, user} <- Accounts.register_user(changeset.changes) do
+      conn
+      |> put_view(UserRegistrationView)
+      |> render("show.json", %{data: user})
+    else
+      {:error, %Ecto.Changeset{} = _changeset} ->
```

Comment:
should we move this error to `fallback controller` @nvminhtue 🤔

Reply:
Updated in [4fc3f20](https://github.com/nvminhtue/elixir_internal_certificate/pull/42/commits/4fc3f20c5b2b4369b4ca39bf951d756d7bc7f928)